client/app/shared/ci-helper/ci-helper.controller.js

```diff
@@ -514,6 +514,25 @@ const CiHelperController = function CiHelperController(
         });
     };
 
+    $scope.setFavouriteLauncher = function (launcher) {
```

Comment:
Please add as a separate function and make as a method of the controller (`vm`)

Reply:
fixed